oracle/Cargo.toml

```diff
@@ -12,6 +12,9 @@ ff-fft = { path = "../zexe/ff-fft", features = [ "parallel"] }
 ocaml = { version = "0.18.1", optional = true }
 rand = "0.7.3"
 rayon = { version = "1" }
+num-bigint = "0.3"
+custom_derive = ""
```

Comment:
wait, this works o_O?

Reply:
Yeah, I tested it by
* Serializing field element to base10 string using this method
* Deserializing this base10 string back to a field element (using `Fp::from_str`)
* Checked that it's equal to what I started with

I saw that in other places in zexe they used `BigInt::parse_bytes` and `BigInt::from_bytes_le`

Reply:
It's not great, but it gives you what you need to create the OCaml test vectors.

Reply:
I didn't know you could include a library without a version number. It also looks like it was superseded by another library? https://crates.io/crates/custom_derive

Reply:
add version number, make dev dependency & update readme